src/applications/edu-benefits/components/AutosuggestField.jsx

```diff
@@ -0,0 +1,296 @@
+import React from 'react';
```

Comment:
Most of this file comes from [this](https://github.com/usds/us-forms-system/blob/0fc545d47a10c530ee6b944a7040853e828c3d94/src/js/fields/AutosuggestField.jsx).